gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/test/java/io/gravitee/am/gateway/handler/root/resources/handler/loginattempt/LoginAttemptHandlerTest.java

```diff
@@ -124,4 +128,13 @@ public void mustTryLoginAttemptApplied() throws InterruptedException {
         Thread.sleep(1000);
         verify(spyRoutingContext, times(1)).next();
     }
+
+    private SortedSet<ApplicationIdentityProvider> getApplicationIdentityProviders(String... identities) {
+        var set = new TreeSet<ApplicationIdentityProvider>();
```

Comment:
You can do the same operation in one statement:

`return Arrays.stream(identities).map(s -> new ApplicationIdentityProvider(s, -1)).collect(Collectors.toCollection(TreeSet::new));`